src/6model/reprs/P6bigint.c

```diff
@@ -4,30 +4,27 @@
    #define MIN(x,y) ((x)<(y)?(x):(y))
 #endif
 
-/* A forced 64-bit version of mp_get_long, since on some platforms long is
- * not all that long. */
-static MVMuint64 mp_get_int64(MVMThreadContext *tc, mp_int * a) {
-    int i, bits;
+/* Handles both int and uint cases. */
+static MVMuint64 mp_get_int64(MVMThreadContext *tc, mp_int * a, int is_signed) {
```

Comment:
The difference between name and return type did confuse me at first, since it is currently used for both.

I have some follow-on plans to break it out into a `mp_get_uint64` and `mp_get_int64`, should I also rename it now?